arch/arm/include/aarch32/cortex_a_r/tcm.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019, Antmicro
```

Comment:
I believe these should all be 2020 now

Reply:
should be fixed now (here and in all the other occurrences)